src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -1746,10 +1726,8 @@ ssize_t sockinfo_tcp::rx(const rx_call_t call_type, iovec* p_iov, ssize_t sz_iov
 
 	si_tcp_logfunc("rx completed, %d bytes sent", total_rx);
 
-#ifdef VMA_TIME_MEASURE
 	if (0 < total_rx)
-		TAKE_T_RX_END;
-#endif
+		TAKE_T_RX_END; // VMA_TIME_MEASURE
```

Comment:
Please verify that this doesn't impact performance as you now do the `if` test regardless of whether `VMA_TIME_MEASURE` is defined or not.